nuklear.h

```diff
@@ -26423,6 +26423,7 @@ nk_textedit_key(struct nk_text_edit *state, enum nk_keys key, int shift_mod,
          if (shift_mod) {
             nk_textedit_prep_selection_at_cursor(state);
             state->cursor = state->select_end = state->string.len;
+			if (state->select_start >= state->select_end) state->select_start = 0;
```

Comment:
Would you mind adding this to nuklear_text_editor.c too? https://github.com/Immediate-Mode-UI/Nuklear/blob/6e80e2a646f35be4afc157a932f2936392ec8f74/src/nuklear_text_editor.c#L654-L658

Otherwise, we'll lose the change next time nuklear.h is updated.

Reply:
OK

I focused on solving the cause of the problem. In this last commit, I limited the cursor position to the text size prevents the visual bug of the cursor going to the beginning of the editbox, and its consequences.

Reply:
Thanks so much! If you'd like some help, I'd be happy to make the update.